templates/deno-ts/package.json

```diff
@@ -1,20 +1,20 @@
 {
-    "private": true,
-    "sideEffects": false,
-    "scripts": {
-      "build": "cross-env NODE_ENV=production remix build",
-      "start": "cross-env NODE_ENV=production deno run --allow-net --allow-read --allow-env ./build/index.js",
-      "dev": "cross-env NODE_ENV=development remix build && run-p dev:*",
-      "dev:remix": "cross-env NODE_ENV=development remix watch",
-      "dev:deno": "cross-env NODE_ENV=development deno run --watch --allow-net --allow-read --allow-env ./build/index.js"
-    },
-    "devDependencies": {
-      "@remix-run/dev": "*",
-      "cross-env": "^7.0.3",
-      "npm-run-all": "^4.1.5"
-    },
-    "engines": {
-      "node": ">=14"
-    }
-  }
-  
\ No newline at end of file
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "cross-env NODE_ENV=production remix build",
+    "start": "cross-env NODE_ENV=production deno run --allow-net --allow-read --allow-env ./build/index.js",
+    "dev": "cross-env NODE_ENV=development remix build && run-p dev:*",
+    "dev:remix": "cross-env NODE_ENV=development remix watch",
+    "dev:deno": "cross-env NODE_ENV=development deno run --watch --allow-net --allow-read --allow-env ./build/index.js"
+  },
+  "devDependencies": {
+    "@remix-run/dev": "*",
+    "cross-env": "^7.0.3",
+    "npm-run-all": "^4.1.5"
+  },
+  "engines": {
+    "node": ">=14"
+  },
+  "name": "remix-template-deno-ts"
```

Comment:
@mcansh I think running `npx sort-package-json` on this file would be a good idea.
This way the file in our repo is the same as the file you get when running `npx create-remix@latest`

Reply:
we do this during create-remix fwiw, but might not be a bad idea to get them all cleaned up before that